.eslintrc.js

```diff
@@ -35,6 +36,7 @@ module.exports = {
         "id-blacklist": 0,
         "operator-linebreak": 0,
         "no-mixed-operators": 0,
+        "no-shadow": "off",
```

Comment:
bug with typescript: https://stackoverflow.com/questions/63961803/eslint-says-all-enums-in-typescript-app-are-already-declared-in-the-upper-scope

Reply:
Nice, but I'd hesitate to make this change if we don't need it right now (see below), since bumping a version later on should suffice.

Reply:
https://typescript-eslint.io/rules/no-shadow/